⬆️ Fix lost theme settings that causeing 500 / 419 errors in some cases

duckietm opened the pull request:
This will fix the following error:

resources/themes//css/app.scss. {"view":{"view":"/var/www/Test-CMS/resources/themes/atom/views/layouts/app.blade.php","data":[]},"exception":"[object] (Spatie\\LaravelIgnition\\Exceptions\\ViewException(code: 0): Unable to locate file in Vite manifest: resources/themes//css/app.scss. at /var/www//Test-CMS/vendor/laravel/framework/src/Illuminate/Foundation/Vite.php:988)

As in the current state the cache will be hardcoded 5 min. and this way it will cache forever unless a change is made

ObjectRetros commented on app/Helpers/helper.php:
```diff
@@ -12,7 +12,13 @@ function setting(string $setting): string
 }
 
-function permission(string $permission): string
+function permission(string $permission): string|bool
```
```suggestion
function permission(string $permission): string|int
```

ObjectRetros committed:
merge main into dev (#52)

duckietm committed:
🆙 Fix cacheing for websettings

This will fix the following error:

resources/themes//css/app.scss. {"view":{"view":"/var/www/Test-CMS/resources/themes/atom/views/layouts/app.blade.php","data":[]},"exception":"[object] (Spatie\\LaravelIgnition\\Exceptions\\ViewException(code: 0): Unable to locate file in Vite manifest: resources/themes//css/app.scss. at /var/www//Test-CMS/vendor/laravel/framework/src/Illuminate/Foundation/Vite.php:988)

As in the current state the cache will be hardcoded 5 min. and this way it will cache forever unless a change is made

ObjectRetros commented:
@duckietm I don't quite understand this PR, can you please elaborate?

ObjectRetros commented:
Closing as it should be fixed in:
https://github.com/ObjectRetros/atomcms/commit/964226f5162a80501491f2f8a7d92269c60eb382
https://github.com/ObjectRetros/atomcms/commit/ad05779a574ced1d2e89d297289b39afef725d90

Thank you for the contribution!
